skins/laika/src/components/pages/donation_confirmation/SummaryLinks.vue

```diff
@@ -1,12 +1,19 @@
 <template>
 	<div class="donation-links is-hidden-print">
 		<a id="print-link" href="javascript:window.print()">{{ $t( 'donation_confirmation_print_confirmation' ) }}</a>
-		<a id="comment-link" :href="donationCommentUrl" v-if="donationCanBeCommented">{{ $t( 'donation_confirmation_comment_button' )
-			}}</a>
+		<a id="comment-link" @click="openPopUp()" v-if="donationCanBeCommented" :disabled="disableCommentLink">
+			{{ disableCommentLink ? $t( 'donation_comment_popup_thanks' ) : $t( 'donation_confirmation_comment_button' ) }}
+		</a>
+		<b-modal :active.sync="addComment" has-modal-card>
```

Comment:
Currently the page does a bit of jump if you click the donation comment link. I looked into it and if you add `scroll="keep"` to the modal properties, the page will not "jump to the side" anymore. Taken from the docs: https://buefy.org/documentation/modal/#api-view